apps/agent/src/main/api.ts

```
@@ -0,0 +1,80 @@
+	constructor() {
+	}
+
+	get getToken() {
+		const auth = getAuthConfig();
+		const token = auth.token;
+		return token;
+	}
+
+	get defaultHeaders() {
+		return {
+			'Authorization': 'Bearer ' + this.getToken,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)